be/src/olap/push_handler.h

```diff
@@ -209,6 +216,44 @@ class LzoBinaryReader: public IBinaryReader {
     size_t _next_row_start;
 };
 
+class PushBrokerReader {
+public:
+    PushBrokerReader() 
+        : _ready(false),
+          _eof(false) {}
+    ~PushBrokerReader() {}
+
+    OLAPStatus init(const Schema* schema,
+                    const TBrokerScanRange& t_scan_range,
+                    const TDescriptorTable& t_desc_tbl);
+    OLAPStatus next(ContiguousRow* row);
+    void print_profile();
+
+    OLAPStatus finalize() {
```

Comment:
I think for reader, `close` maybe better than `finalize`.

Reply:
ok